tasks/docs.rake

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+
+task 'docs:clobber' do
+  FileUtils.remove_dir('api-docs') if Dir.exist?('api-docs')
+end
+
+task 'docs:set-env' do
+  ENV['DOCSTRINGS'] = '1'
+  ENV['SITEMAP_BASEURL'] = 'http://docs.aws.amazon.com/sdk-for-ruby/v3/api/'
+  ENV['BASEURL'] = 'http://docs.aws.amazon.com/'
+end
+
+# return a list of all of the other SDK gems this gem depends on
+# sdk gems are only those in the /gems folder that are authored here
+def sdk_dependencies(sdk_gem)
```

Comment:
Will this also build docs for core and sigv4?

Reply:
Yup!  And for gems like s3, it includes kms.